tests/update/patch_before_migration.pm

```diff
@@ -33,7 +33,7 @@ sub patching_sle {
 
     assert_script_run("zypper lr && zypper mr --disable --all");
     save_screenshot;
-    yast_scc_registration();
+    sle_register("R");
```

Comment:
How about rename sle_register to sle_ncc_register or just ncc_register?
And how about hash argument here like: action => 'register' (This can be default)  and action => 'unregister', I think easier to understand this way.

Reply:
Actually, this part contain both scc register and ncc register. I suppose sle_register is more clearly and suitable here.

Reply:
OK, it's fine. :)